build/packages.json

```diff
@@ -1,5 +1,6 @@
 {
     "requiredNodeMajorVersion": 16,
+    "requiredNPMMajorVersion": 8,
```

Comment:
We currently support NPM 7.x - why make it 8? In my understanding, Node.js ships with NPM 7.x, so it would be a better out-of-the-box experience if people only have to install whatever Node.js version we tell them to, not manually upgrade NPM afterwards.

Reply:
Mainly because of #597. That `overrides` only works in `npm` 8.

Reply:
Oh, good to know! I'd argue you could first ship this PR with `requiredNPMMajorVersion` set to `7`, and then, in #597, increase it to `8`. Seems most clean to me, but I wouldn't be anal about that :D

More importantly: Looking at #597 - if I read you correctly, we will now require >=8.3, not >=8.0 -> So don't we need some logic here for specifying not just the required _major_ version, but also the _minor_ version?

I guess the most simple approach would be to just... um... right pad the value supplied here, e.g.:

`requiredNodeVersion: 16` -> `16.0.0`
`requiredNodeVersion: 16.1` -> `16.1.0`
`requiredNodeVersion: 16.1.2` -> `16.1.2`

But I'd be totally fine with just writing the full version out, e.g.: `requiredNodeVersion: 16.0.0`.

Technically, we could stick to `requiredNodeMajorVersion` but only change `requiredNPMMajorVersion` to `requiredNPMVersion`, but seems like we can support it for both while at it.

Lastly, I promise, I'm also thinking we could be more clear here on how we use these values, with the naming. For example: `minimumRequiredNodeVersion`. Don't love that particular name, but I guess you get the idea. Totally optional though.

Reply:
Well, it should work for 8.0, actually, but somehow I wasn't able, maybe because the configuration was not 100% correct. I'll change the major to 7 meanwhile, and to the other changes.